src/frontend/src/lib/components/tokens/TokensMenu.svelte

```diff
@@ -28,16 +26,10 @@
 	<div class="flex flex-col gap-3">
 		<TokensZeroBalance />
 
-		{#if $networkICP || $networkEthereum}
-			<div class="my">
-				<Hr />
-			</div>
+		<div class="my">
+			<Hr />
+		</div>
 
-			{#if $networkICP}
-				<IcManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
-			{:else if $networkEthereum}
-				<ManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
-			{/if}
-		{/if}
+		<IcManageTokensMenuButton on:icCloseMenu={() => (visible = false)} />
```

Comment:
This component will be renamed (to remove the Ic prefix) and moved in another PR I guess?

Reply:
correct!

Reply:
created PR #1523